earthly/rust/Earthfile

```diff
@@ -77,10 +79,18 @@ rust-base:
 
     # Ensure we have all the necessary targets
     RUN rustup target add wasm32-unknown-unknown
-    RUN rustup target add wasm32-wasip1 # wasm32-wasip2 not yet available
+    RUN rustup target add wasm32-wasip1
+    # RUN rustup target add wasm32-wasip2 # wasm32-wasip2 not yet available in stable
 
     RUN rustup target add wasm32-unknown-unknown --toolchain nightly
-    RUN rustup target add wasm32-wasip1 --toolchain nightly # wasm32-wasip2 not yet available
+    RUN rustup target add wasm32-wasip1 --toolchain nightly 
+    RUN rustup target add wasm32-wasip2 --toolchain nightly
```

Comment:
I want this in here because I want to know the tooling is progressing through the pipeline.
rust 1.81 will have `wasm32-wasip2` in stable, and so we should also have it in nightly (it's already in nightly).
WasiP2 is the wasm target we ultimately want to support with rust, so we don't need the wasi component adapter for rust projects (only Go/C/etc until they too support wasi-p2 natively).

Reply:
Oh, so sorry, I just though it was a duplicate, my bad